### ISSUE TITLE
Closes after a few minutes

### ISSUE THREAD

**Issue by hankypoo7** [Blobby-Boi/ExtHang3r#5]
I run it and it works perfectly fine, but after maybe 15-30 minutes it stops working. Any fix?

**Comment by Blobby-Boi** [comment]
Try keeping the tab open some people say that has worked for them

**Comment by hankypoo7** [comment]
@Blobby-Boi ok I'll let you know if it works

**Comment by hankypoo7** [comment]
So far it's working, although my Chromebook is kinda slow. also, you have any way to install extensions when my school blocked it?

**Comment by Blobby-Boi** [comment]
Did they block the webstore?

**Comment by hankypoo7** [comment]
no but they blocked certain extensions

**Comment by Rosiiiie** [comment]
I'm having this same issue. For example, say I kill Lightspeed Filter (an extension which restricts the Chromebooks from accessing sites). After ~15 minutes, Lightspeed comes back. It doesn't matter if I keep the ExtHang3r tab open in the background or if it's off to the side, it'll just stop working.

**Comment by Pcawsomecrafter** [comment]
same also some websites are blocked and some are not

**Comment by Rosiiiie** [comment]
While testing again to see if I can get the tab to not refresh , this happened . My thought is , after 15-or-so minutes , the ExtHang3r tab will refresh and open the icon for the extension you're trying to block , which breaks the exploit .
![image](https://github.com/user-attachments/assets/d8b9f3af-2409-4c8a-926b-217597ab8399)

**Comment by hankypoo7** [comment]
@Rosiiiie yeah that's what I was thinking

**Comment by Pcawsomecrafter** [comment]
so is there a way to block it forever. also is there a way to bypass the second firewall

**Comment by Rosiiiie** [comment]
I may have found a temporary fix until we can (hopefully) get a legitimate fix.

What I did was I had two ExtHang3r tabs open and ran the exploit. After the first tab breaks, quickly run the exploit on the second tab.

**Comment by Pcawsomecrafter** [comment]
but it laggy

**Comment by Rosiiiie** [comment]
I agree , it does make the Chromebooks super slow , but it _does_ work , I'm able to get through entire games of Fortnite without much of a problem

**Comment by Pcawsomecrafter** [comment]
also kinda unrelated but is there a way to get past the school wifi restrictions without a hotspot or bookmarklets

**Comment by mooshymango** [comment]
Yep having the same issue of a Chromebook not keep waiting for the program and send the site to the image. Haven’t found a fix yet unless you just stay on the site and keep clicking wait when chrome asks you if you want to keep waiting or quit.

**Comment by jeffrey6576i** [comment]
i spent 2 years tryna find extensions...

**Comment by Chattyice234** [comment]
I can't say much because my school uses GoGuardian, but exthang3r isn't for deleting extensions permanently. use the time the extension is killed wisely for whatever you need, I'd say it's for small tasks like uploading something to Mediafire, neocities, whatever. 15-30 minutes is a good time.
